spec/controllers/two_factor_authentication/otp_expired_controller_spec.rb

```diff
@@ -61,6 +61,55 @@
     end
 
     context 'otp expired redirect feature is turned off' do
+      it 'does not redirect' do
+        allow(IdentityConfig.store).to receive(:allow_otp_countdown_expired_redirect).
+          and_return(false)
+
+        get :show
+
+        expect(response).to_not render_template(:show)
+      end
+    end
+
+    context 'with a new account' do
+      before do
+        allow(controller).to receive(:new_account_mfa_registration?).and_return(true)
+        allow(controller).to receive(:unconfirmed_phone?).and_return(true)
```

Comment:
Since the logic for these happens in the controller, I think it'd be better if we let the real code run, and set up / stub the underlying session values in such a way that they return the expected value. Otherwise we have no test coverage for the logic of these methods.